library/src/permutation/permutation_meta_traits.hpp

```diff
@@ -47,21 +47,19 @@ namespace hiptensor
               typename Scale,
               ck::index_t NumDim>
     struct MetaTraits<ck::tensor_operation::device::DeviceElementwise<
-                                                                    InDataTypeTuple,
-                                                                    OutDataTypeTuple,
-                                                                    Aop,
-                                                                    Bop,
-                                                                    Scale,
-                                                                    NumDim>>
+        InDataTypeTuple,
+        OutDataTypeTuple,
+        ck::tensor_operation::element_wise::UnaryCombinedOp<Aop, Scale, Bop>,
+        NumDim>>
     {
         constexpr static ck::index_t NDim = NumDim;
 
-        using InDataT   = InDataTypeTuple;
-        using OutDataT  = OutDataTypeTuple;
-        
-        using AOp = Aop;
-        using BOp   = Bop;
-        using ScaleOp   = Scale;
+        using InDataT  = InDataTypeTuple;
+        using OutDataT = OutDataTypeTuple;
+
+        using AOp     = Aop;
+        using BOp     = Bop;
+        using ScaleOp = Scale;
```

Comment:
Might be good to also keep track of the combined op, e.g., using CombinedOp = ck::tensor_operation....